{% extends "base_second.html" %}

{% block extrahead %}
	<!--This two was missed in base-->
{% endblock %}

{% block content %}

      <!-- ============================================-->
      <!-- Preloader ==================================-->
      <div class="preloader" id="preloader">
        <div class="loader">
          <div class="line-scale-pulse-out-rapid">
            <div></div>
            <div></div>
            <div></div>
            <div></div>
            <div></div>
          </div>
        </div>
      </div><!-- ============================================-->
      <!-- End of Preloader ===========================-->

      <!-- ============================================-->
      <!-- <section> begin ============================-->
      <section class="border-top border-300" id="default-process">
        <div class="container">
          <div class="row justify-content-center text-center mb-6">
            <div class="col-lg-6">
              <h2 class="fs-4 text-underline mb-3">Our Discoveries</h2>
              <p class="text-500 font-italic">
				We are a group of highly motivated and creative people, discovering and innovating together 
				to make precision medicine a reality through the catalytic power of big genomic data, 
				statistical genetics, and artificial intelligence.
 				
			  </p>
              <hr class="hr-short mt-5 border-300" />
            </div>
          </div>
          <div class="row">
            <div class="col-12 pl-lg-7">
              <div class="row align-items-end">
                <div class="col-lg-6 order-lg-2 text-center pb-sm-2 pb-lg-6"><img class="img-fluid" src="../pages/assets/img/illustration/process-04.svg" alt="" width="400" /></div>
                <div class="col-lg-6 border-lg-left border-lg-bottom border-300 pb-lg-3 mt-4 mt-lg-0 mb-8 mb-lg-0">
                  <div class="process-item ml-6 ml-sm-8 ml-lg-6"><span class="process-item-number">01</span>
                    <h3>Disease risk prediction by polygenic risk scores</h3>
                    <p>
						
					Genome-wide polygenic scores – which integrate information from many common DNA variants into a single continuous measure of inherited risk – 
					have shown striking ability to stratify risk for incident and prevalent diseases in recent studies, mostly in European ancestry. 
					Whether this concept can be effectively extended to other ethnic populations, e.g., South Asians, has remained uncertain. 
					We derived and tested a genome-wide polygenic score tuned specifically to individuals of South Asian ancestry. 
					Our results confirm a robust association of a new genome-wide polygenic score with the risk of coronary artery disease, 
					with strikingly consistent results across South Asians living in the United Kingdom, Bangladesh, and India 
					<i>(<strong>Wang</strong> et al. 2020, J Am Coll Cardiol. )</i>. In other studies, 
					our results show that polygenic scores can not only be used to predict the second event among people already with a high risk of coronary artery disease
					 <i>(Emdin*, Bhatnagar*, <strong>Wang*</strong> et al. 2020, Circ Genomic Precis Med.)</i>, 
					 but also can be used to predict the genetic level of disease biomarkers and provide help in enriching effective samples for clinical trials 
					 <i> (Dron*, <strong>Wang*</strong> et al. 2021, Circ Genomic Precis Med.)</i>.
			 			
                    </p>
                  </div>
                </div>
              </div>
              <div class="row">
                <div class="col-lg-6 border-lg-right border-lg-bottom border-300 text-center py-lg-8"><img class="img-fluid" src="../pages/assets/img/illustration/process-01.svg" alt="" width="400" /></div>
                <div class="col-lg-6 mt-4 mt-lg-0 mb-8 mb-lg-0 my-lg-8">
                  <div class="process-item ml-6 ml-sm-8 ml-lg-6"><span class="process-item-number">02</span>
                    <h3>Genetic modifier discovery for common and rare diseases </h3>
                    <p>
						
						Not all people carrying disease-causing mutations or high-risk genetic variants develop the disease. 
						Thus, identifying the genetic and non-genetic modifiers is important to understand the disease mechanisms and build useful and accurate statistical models for disease risk prediction and disease prevention. 
						By genetic admixture-mapping and analyzing gene expression data, we identified the UBD gene as a genetic modifier of APOL1 related kidney disease 
						(two high-risk APOL1 disease risk alleles increased the risk about ten times in a recessive form). 
						Following that, cell-based experiments showed that the UBD and APOL1 protein has physical interaction and that UBD expression mitigates APOL1-mediated cell death
						<i>(Zhang*, <strong>Wang*</strong> et al. 2018, PNAS)</i>. 
						On the other hand, illustrated by three diseases (coronary artery disease, breast cancer, and colorectal cancer) and by modeling the rare monogenic disease-causing mutations in the polygenic risk background, 
						we show that the polygenic risk has a significant impact on the disease penetrance in individuals carrying the monogenic mutations, 
						in some cases bringing risk closer to the population average<i>(Fahed*, <strong>Wang*</strong>, homburger*, et al. 2020, Nat. Commun)</i>.
						 More importantly, this model is generally applicable to many other conditions. The integration of the risk conferred by monogenic mutations with polygenic backgrounds has the opportunity to provide more accurate risk prediction and enable precise patient stratification and management.
						
					</p>
				</div>
                </div>
              </div>
              <div class="row">
                <div class="col-lg-6 order-lg-2 text-center align-self-center"><img class="img-fluid" src="../pages/assets/img/illustration/process-02.svg" alt="" width="400" /></div>
                <div class="col-lg-6 border-lg-left border-lg-bottom border-300 py-lg-8 mt-4 mt-lg-0 mb-8 mb-lg-0">
                  <div class="process-item ml-6 ml-sm-8 ml-lg-6"><span class="process-item-number">03</span>
                    <h3>Genetic diagnosis, disease risk gene discovery,  and drug target discovery and optimization</h3>
                    <p>
						We studied whole-exome sequencing data of more than 400 focal segmental glomerulosclerosis (FSGS, a rare form of kidney disease)
						 families and 600 ancestry-matched controls. From this study, 
						 we 1) dissected the genetic architecture of FSGS from the common and rare variant perspective, 
						 2) cataloged the rare variants in genes known to cause FSGS disease to estimate the rate of genetic diagnosis, 
						 3) analyzed the disease-causing mutation pattern of protein domain enrichment for advancing the understanding of disease mechanisms, 
						 4) developed new statistical models to discover potential new genes that increase the disease risk of FSGS <i>(<strong>Wang</strong> et al. 2019, JASN )</i>. 
						 Meanwhile, we also created a novel visualization method for presenting large family pedigrees in a small printable area 
						 <i>(Chun*, <strong>Wang*</strong> et al. 2020, KI Reports)</i>. On the drug optimization arm, we analyzed the genomic sequencing data of about 50,000 people from the UK Biobank, 
						 distinguished the side effects of Volanesorsen (the only FDA-approved treatment for familial chylomicronemia syndrome)
						  is because of the drug-specific effects, not because of the impact of the target gene <i>(Khetarpal*, <strong>Wang*</strong> et al. 2019, N. Engl. J. Med)</i>. 
						  This clarification of the cause of the drug’s side effects is crucial for drug optimization as it paved the theoretical pathway for drug optimization. 
						  I am also analyzing the whole-genome/exome sequencing data of about 258,000 participants (41,000 coronary artery disease cases and 217,000 controls) to discover novel coronary artery disease risk genes using rare variant burden tests (results in submission).

						</p>
					</div>
                </div>
              </div>
              <div class="row overflow-hidden pb-1">
                <div class="col-lg-6 border-lg-right border-300 text-center py-lg-8"><img class="img-fluid" src="../pages/assets/img/illustration/process-03.svg" alt="" width="400" /></div>
                <div class="col-lg-6 align-self-center mt-4 mt-lg-0 mt-lg-8">
                  <div class="process-item process-item-last ml-6 ml-sm-8 ml-lg-6"><span class="process-item-number">04</span>
                    <h3>Method development and statistical modeling</h3>
                    <p>
						
						Identifying genes that have been positively selected in human evolution can not only help us understand the adaptive evolution of human beings, 
						but also help understand the risk mechanism of diseases. Although thanks to the rapid development of next-generation gene sequencing and genotyping technology, 
						a large amount of genetic data has been rapidly generated in recent years. However, 
						genetic testing methods for accurately and efficiently analyzing and identifying genes subject to positive selection from these data are still very limited.
						Furthermore, it is even more challenging to pinpoint the driver mutation that was selected. 
						We, therefore, designed a fast and accurate method for detecting positive selection genes and locating positive selection driving mutations. 
						This algorithm is comparable in statistical power to the most effective and most commonly used algorithm(iHS) at that time, but the calculation speed is more than 10,000 times faster.
						More importantly, the accuracy of the new algorithm for locating positive selection effect sites is significantly better than that of the iHS algorithm. 
						In addition, when the selection effect is relatively weak, the positioning ability of the new algorithm far exceeds other traditional algorithms 
						<i>(<strong>Wang</strong> et al. 2014, Mol Biol Evol, and He, <strong>Wang</strong> et al. 2015, Genome Research)</i>.
						
					</p>
					</div>
                </div>
              </div>
            </div>
          </div>
        </div><!-- end of .container-->
      </section><!-- <section> close ============================-->
      <!-- ============================================-->


{% endblock %}
